ompi/communicator/comm.c

```diff
@@ -1509,9 +1509,6 @@ static int ompi_comm_idup_with_info_activate (ompi_comm_request_t *request)
 
 static int ompi_comm_idup_with_info_finish (ompi_comm_request_t *request)
```

Comment:
I'd prefer to leave this in case something changes later on where we actually need this function to do something - note its appeaded to the schedule for a comm constructor at line 1505 of comm.c